sormas-api/src/main/resources/validations.properties

```diff
@@ -107,6 +107,11 @@ specifyYear = You have to specify the year
 specifyEpiWeek = You have to specify the epi week
 userNameNotUnique = User name is not unique!
 validCaseContactOrEventParticipant=You have to specify a valid case or contact from the SORMAS database
+validExistingCase=The database does not contain a case with this UUID.
```

Comment:
Please change the names to `noCaseWithUuid`, `noContactWithUuid` etc. - afterwards the PR can be merged :)